src/lib/components/Funding/NoAwardResults.js

```diff
@@ -0,0 +1,39 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 CERN.
+// Copyright (C) 2021 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Message } from 'semantic-ui-react';
+import { i18next } from '@translations/i18next';
+
+export function NoAwardResults({ switchToCustom }) {
+  return (
+    <Message
+      warning
+      icon="warning circle"
+      header={i18next.t('No results found')}
+      content={
+        <p>
```

Comment:
HTML tags are usually not needed when using semantic UI. Needs to be changed probably